website/docs/bestpractices/analytics/trino-best-practices.md

```diff
@@ -0,0 +1,642 @@
+---
+sidebar_position: 2
+sidebar_label: Trino on EKS Best Practices
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CollapsibleContent from '../../../src/components/CollapsibleContent';
+
+# Trino on EKS Best Practices
+[Apache Trino](https://trino.io/) deployment on [Amazon Elastic Kubernetes Service](https://aws.amazon.com/eks/) (EKS) delivers distributed query processing with cloud-native scalability. Organizations can optimize costs by selecting specific compute instances and storage solutions that match their workload requirements while they  combine the power of Trino with the scalability and flexibility of EKS using [Karpenter](https://karpenter.sh/).
+
+This guide provides prescriptive guidance for deploying Trino on EKS. It focuses on achieving high scalability and low cost through optimal configurations, effective resource management, and cost-saving strategies. We cover detailed configurations for popular file formats such as Hive and Iceberg. These configurations ensure seamless data access and optimize performance. Our goal is to help you set up a Trino deployment that is both efficient and cost-effective.
+
+We have a [deployment-ready blueprint](https://awslabs.github.io/data-on-eks/docs/blueprints/distributed-databases/trino) for deploying Trino on EKS, which incorporates the best practices discussed here.
+
+Refer to these best practices for the rational and further optimization/fine-tuning.
+
+<CollapsibleContent header={<h2><span>Apache Trino</span></h2>}>
+
+## Basics
+
+Apache Trino is a powerful distributed SQL query engine designed for high-performance analytics and big data processing.
+
+### Core Architecture
+
+- Distributed coordinator-worker model
+- In-memory processing architecture
+- MPP (Massively Parallel Processing) execution
+- Dynamic query optimization engine
+
+### Key Capabilities
+
+- Query Federation
+- Simultaneous queries across multiple data sources
+- Support for heterogeneous data environments
+- Real-time data processing capabilities
+- Unified SQL interface for diverse data sources
+
+### Connectors Ecosystem
+
+- 50+ production-ready connectors including:
+  - Cloud storage (AWS S3)
+  - Relational databases (PostgreSQL, MySQL, SQL Server)
+  - NoSQL stores (MongoDB, Cassandra)
+  - Data lakes (Apache Hive, Apache Iceberg, Delta Lake)
+  - Streaming platforms (Apache Kafka)
+
+### Performance Optimizations
+
+- Advanced cost-based optimizer
+- Dynamic filtering
+- Adaptive query execution
+- Sophisticated memory management
+- Columnar processing support
+
+### Use Cases
+
+- Interactive analytics
+- Data lake queries
+- ETL processing
+- Ad-hoc analysis
+- Real-time dashboards
+- Cross-platform data federation
+
+</CollapsibleContent>
+
+<CollapsibleContent header={<h2><span>EKS Cluster Setup</span></h2>}>
```

Comment:
Better to use other words than setup. `Configuration` maybe? This applies to other sections as well.

Reply:
Noted will add in the next iteration